src/fragments/lib/auth/native_common/getting_started/common.mdx

```diff
@@ -67,19 +68,54 @@ import flutter5 from "/src/fragments/lib/auth/flutter/getting_started/12_amplify
 <Fragments fragments={{flutter: flutter5}} />
 
 </Block>
-<Block name="Existing resources">
-Existing Authentication resources from AWS (e.g. Amazon Cognito UserPools or Identity Pools) can be used with the Amplify Libraries by running:
+
+<Block name="Amplify CLI (Import)">
+
+> Prerequisites: [Install and configure](/lib/project-setup/prereq/q/platform/flutter/#install-and-configure-the-amplify-cli) the Amplify CLI in addition to the Amplify libraries and [necessary dependencies](/lib/project-setup/create-application/q/platform/flutter/#2-install-amplify-libraries).
+
+To import existing Amazon Cognito resources into your Amplify project, **execute the command**:
 
 ```bash
 amplify import auth
 ```
+```console
+? What type of auth resource do you want to import?
+   Cognito User Pool and Identity Pool
+   Cognito User Pool only
+```
+
+Once you've selected an option, you'll be able to search for and import an existing Cognito User Pool and Identity Pool (or User Pool only) within your AWS account.  The `amplify import auth` command will also do the following:
+- Automatically populate your Amplify Library configuration files (`amplifyconfiguration.dart` and `amplifyconfiguration.json`) with your chosen Amazon Cognito resource information
+- Provide your designated existing Cognito resource as the authentication & authorization mechanism for all auth-dependent categories (API, Storage and more)
+- Enable Lambda functions to access the chosen Cognito resource if you permit it
+
+> If you have previously enabled an Amplify category that uses Auth behind the scenes (e.g. API category), run `amplify update auth` command to edit your configuration. 
+
+After configuring your Authentication options, update your backend and deploy the service by running the `push` command:
+
+```bash
+amplify push
+```
 
+Now, the authentication service has been deployed and you can start using it. To view the deployed services in your project at any time, go to Amplify Console by running the following command:
+
+```bash
+amplify console
+```
 For more details, see how to [Use an existing Cognito User Pool and Identity Pool](/cli/auth/import).
 
-If you are not using the Amplify CLI, a Cognito User Pool and Identity Pool can be used by referencing them in your `amplifyconfiguration.json` file.
+</Block>
+
+<Block name="Manual Configuration">
```

Comment:
The switcher looks great and clear. Thanks for the update 👍

The content of `common.mdx` is rendered for amplify-swift and amplify-android libraries documentations as well. Might need to create platform specific content and import per platform here. Example [here](https://github.com/aws-amplify/docs/blob/88915b19fc63858cadac2d34597c1c0c99f94889/src/fragments/lib/auth/existing-resources.mdx?plain=1#L10-L15).